common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -422,13 +433,8 @@ protected CorruptedBarOverlay() {
         }
 
         @Override
-        public float textureHeight() {
-            return Texture.UNIVERSAL_BAR.height() / 2f;
-        }
-
-        @Override
-        public WynnBossBarUtils.BarProgress progress() {
-            return WynnBossBarUtils.getCorruptedBar();
+        public BossBarModel.BarProgress progress() {
+            return BossBarModel.corruptedBar.getBar();
```

Comment:
`getBar()` sounds like it should be `getBarProgress` or even just `getProgress`. (Changes needed in many places, I just commented it here)

Reply:
I've elected to go with `getBarProgress` as `getProgress` already exists.